### ISSUE TITLE
how far along is the project?

### ISSUE THREAD

**Issue by Granolledevoreurdemontagne** [Spore-Multiplayer/Spore-Multiplayer#8]
how far along is the project?

**Comment by Studumb** [comment]
> how far along is the project?

Still under development, nothing has gone so far

**Comment by LoLiPoP89** [comment]
Hello. Is there any information, is the project being developed?

**Comment by Studumb** [comment]
> Hello. Is there any information, is the project being developed?

Yes, it is still under development, there also a community you can join here
https://discord.gg/fMXtW2fsPe
